openlibrary/templates/lib/dropper.html

```diff
@@ -0,0 +1,17 @@
+$def with(primary_button, dropdown_content, additional_classes='')
```

Comment:
This needs comments.  Would be good to know that passing `generic-dropper--disabled` as an additional class will disable the dropper for logged out patrons.